### DISCUSSION TITLE
Depth Camera Access

### DISCUSSION THREAD

**Discussion by utkudotdev** [empriselab/RCareWorld#59] [Help - Programming] [answered]
Hi! For the assisted bathing challenge, I was told that a depth camera is available. I see a camera under ID `654321`. Are we allowed to use this? If so, can we use computer vision libraries like OpenCV?

**Comment by YoruCathy** [comment]
Yep, feel free to use it.

**Reply by utkudotdev** [reply]
Thanks! Is OpenCV also fine? 

Open3D and NumPy came with the environment when I installed it. I'm guessing those are okay?

**Reply by YoruCathy** [reply, marked as answer]
Yep feel free to use any library that’s there and install new ones
